cfme/services/catalogs/orchestration_template.py

```diff
@@ -140,8 +140,10 @@ def create(self, content):
         view = navigate_to(self, "AddTemplate")
         if(self.template_type == "CloudFormation Templates"):
             temp_type = "Amazon CloudFormation"
-        else:
+        elif(self.template_type == "Heat Templates"):
```

Comment:
Good point.  There is no default other than to raise an exception I suppose.  That would probably be best.  This list has been pretty static for a long time.